src/components/Box/Box.tsx

```diff
@@ -0,0 +1,10 @@
+import React from "react"
+
+import styles from "./Box.module.scss"
+
+export const Box: React.FC = ({ children }) => (
+  <div className={styles.box}>
+    <div className={styles.shadow}></div>
```

Comment:
This is probably a bit "hacky" but I couldn't find a way to make it look similar to the designs by just using `box-shadow`. Suggestions are welcome though.

Reply:
I did the same thing for the CTF, absolute positioning the shadow. If it works, it's not stupid haha

Reply:
haha love that